packages/frontend/src/Build.spec.ts

```diff
@@ -418,6 +419,9 @@ test('Do not show an image if it has no repotags and has isManifest as false', a
   vi.mocked(bootcClient.listBootcImages).mockResolvedValue(mockedImages);
   vi.mocked(bootcClient.buildExists).mockResolvedValue(false);
   vi.mocked(bootcClient.checkPrereqs).mockResolvedValue(undefined);
+
+  const linkSpy = vi.spyOn(bootcClient, 'openLink');
```

Comment:
curious why we spy vs using  `vi.mocked(bootcClient.openLink)`line 448

Reply:
Was just copying code from elsewhere in the class. Fixed, will put a separate PR later to fix the other spys.